webAPI.py

```diff
@@ -2,61 +2,125 @@
 import os
 from datetime import datetime
 from flask import Flask, request, jsonify
-from werkzeug.utils import secure_filename
-import glob
 import json
 
 webAPI = Flask(__name__)
 
 UPLOAD_FOLDER = 'uploads'
+PROCESSED_FOLDER = 'processed'
+OUTPUT_FOLDER = 'outputs'
 webAPI.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+webAPI.config['OUTPUT_FOLDER'] = OUTPUT_FOLDER
+webAPI.config['PROCESSED_FOLDER'] = PROCESSED_FOLDER
+ERROR = 400
```

Comment:
With so many Constants, and when some of them repeat themselves, it would make sense to open a python constants file.